.github/workflows/integration.yaml

```
@@ -0,0 +1,48 @@
+name: Integration
+on: [push, pull_request]
+env:
+  GALAXY_TEST_DBURI: 'postgres://postgres:postgres@localhost:5432/galaxy?client_encoding=utf8'
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7, 2.7]
```

```suggestion
        python-version: [3.7]
```
I guess something went wrong somewhere, I though I rebased it into the first commit ...

---

I temporarily dropped it to run the test on 2.7 too.

---

Oops, I was a bit too fast then